etcd.conf.yml.sample

```diff
@@ -155,3 +155,6 @@ cipher-suites: [
 # Limit etcd to specific TLS protocol versions 
 tls-min-version: 'TLS1.2'
 tls-max-version: 'TLS1.3'
+
+# Comma-separated list of feature gate enablement in the format of feature=true|false
+feature-gates: StopGRPCServiceOnDefrag=true,DistributedTracing=false
```

Comment:
`StopGRPCServiceOnDefrag` is false by default in DefaultEtcdServerFeatureGates, we should keep the same value.